### PR TITLE
man: add some basic documentation for sd-boot

### PR DESCRIPTION
**Pull request by keszybz**
I'm sure this can be improved in various ways, but I think it's a good start.

### CODE REVIEW

**Comment by medhefgo** [comment]
- Maybe mention that "timout 0" will result in immediate boot unless a key is held down at boot?
- Boolean options understand true/on and false/off. We shouldn't advertise setting them to 1/0.
- auto-entries (Windows, OSX and EFI shell) and auto-firmware (Reboot to EFI firmware setup) descriptions missing, now that it's merged.
- You got t/T increase/decrease inverted. t increases, T decreases the timeout.
- On that note, we should warn that it will most likely use us keyboard layout, so some oddities such +/- seemingly not working might happen.
- Maybe also mention that arrow keys, Pos1/End, Esc work when editing the cmdline. Alt+d deletes a word. Ctr+w and Alt+Backspace deletes previous word. Ctrl+k clears cmdline.
- Reprint screen is Ctrl+l (it would conflict with l as shortcut for linux)

**Comment by medhefgo** [comment]
And I guess since you merged the video mode setting stuff, they should be mentioned too.

**Comment by keszybz** [comment]
Thanks for the review!

Updated with all the suggested changes and a description of the new loader.conf options.

File: man/loader.conf.xml [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+        editor. The editor should be disabled if the machine can be accessed by
+        unauthorized persons.</para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term>auto-entries</term>
+
+        <listitem><para>Takes a boolean argument. Enable (the default) or disable
+        entries for other boot entries registered as EFI boot variables. In
+        particular, this may useful when loader entries are created to show
```
Comment (medhefgo) [inline man/loader.conf.xml]: may be useful

Also, better would be "other boot entries found on the boot partition"

Comment (keszybz) [reply man/loader.conf.xml]: Updated.

File: man/loader.conf.xml [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+        editor. The editor should be disabled if the machine can be accessed by
+        unauthorized persons.</para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term>auto-entries</term>
+
+        <listitem><para>Takes a boolean argument. Enable (the default) or disable
+        entries for other boot entries found on the boot partition. In particular,
+        this may useful when loader entries are created to show replacement
```
Comment (medhefgo) [inline man/loader.conf.xml]: may _be_ useful

Comment (keszybz) [reply man/loader.conf.xml]: Updated.

**Pull request merged**